windows/devices/uwb/UwbDeviceConnector.cxx

```diff
@@ -739,24 +734,24 @@ UwbConnector::NotificationListenerStop()
 ::uwb::RegisteredCallbackToken*
 UwbConnector::RegisterDeviceEventCallbacks(std::weak_ptr<::uwb::UwbRegisteredDeviceEventCallbacks> callbacks)
 {
-    {
-        std::lock_guard presenceLock{ m_callbacksPresentConditionVariableGate };
-        std::lock_guard writerLock{ m_eventCallbacksGate };
-        m_deviceEventCallbacks = callbacks;
+    std::lock_guard writerLock{ m_eventCallbacksGate };
+    bool isFirstCallback = CallbacksPresent();

```

Comment:
Should this be flipped? If the callbacks are not present (what we're trying to detect), then `CallbacksPresent()` will return `false`, so `!CallbacksPresent() == true`, indicating this would be the first time setting them, right?

```suggestion
    bool isFirstCallback = !CallbacksPresent();
```

Reply:
You're absolutely correct lol